src/Placeholder/connect.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import * as React from 'react';
+
+type Props = {
+  onReady?: boolean,
+  children: React.Node,
+  animate?: string,
+  customAnimate?: () => void,
+};
+
+const connect = (PlaceholderComponent: any) => {
```

Comment:
Could you change the type `any` to something more descriptive? I guess `React.Node` should work.